hokusai/services/deployment.py

```diff
@@ -120,13 +123,14 @@ def current_tag(self):
 
     for deployment in self.cache:
       containers = deployment['spec']['template']['spec']['containers']
-      container_names = [container['name'] for container in containers]
-      container_images = [container['image'] for container in containers]
+      container_images = [container['image'] for container in containers if self.ecr.project_repo in container['image']]
```

Comment:
@ashkan18 here's the fix for the error `Deployment's containers do not reference the same image tag.  Aborting.` you were seeing when running `hokusai pipeline gitcompare`...

Reply:
perfect! 🙌 THANKS!